CMakeLists.txt

```diff
@@ -127,21 +127,17 @@ if(NOT CMAKE_CONFIGURATION_TYPES)
   endif()
 endif()
 
-option(QT6 "Build with Qt6" ON)
-option(QML "Build with QML" OFF)
+set(QT6 ON)
```

Comment:
Yeah, that makes sense.

As I was removing these options, I thought that reverting to Qt5 was unlikely as it would undo a lot of work.

I could keep `QT6` and `QML` and use the QML build option as well as the argument, meaning that a `QML=ON` build would still be able to run with legacy UI if `--qml` is omitted. Does that sound like a better approach?

Reply:
Did you try to compile with Qt6 off and QML off?

Reply:
Mixxx 2.5 will sure be s QT6 only release.
The only reason not to break Qt5 builds is to can go quick back and crosscheck if we have a Qt6 regression or not. So if this is possible without a dedious effort, let's keep it working.

Reply:
With this new push, I try building with `QT6=OFF` (so `QML=OFF`) and it seems to still be working. I didn't do further testing tho, so there may be later asserts/crashes